opcua/api/endpoint.go

```diff
@@ -19,7 +19,7 @@ func browseEndpoint(svc opcua.Service) endpoint.Endpoint {
 			return nil, err
 		}
 
-		nodeID := fmt.Sprintf("ns=%s;%s", req.Namespace, req.Identifier)
+		nodeID := fmt.Sprintf("%s;%s", req.Namespace, req.Identifier)
```

Comment:
Didn't we say that when the req.Namespace and req.Identifier are empty, the defaults of ns=1, i=84 are automatically used, as the user definitely can not know this information.